pkg/kubelet/util/util_windows.go

```diff
@@ -22,6 +22,8 @@ package util
 import (
 	"context"
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
```

Comment:
nit: group imports

```bash
import (
  stdlib

  internal and external packages
)
```

Reply:
Sorry about that, just realized that pretty much every other .go file aside the ones I modified group this nicely. One question though: I'm seeing .go files that have their import section split in 2 (the standard library packages and the rest, as in the first code block below) and others that have 3 (the standard library and 2 others, as in the 2nd code block below). Is there a specific criteria for how to split the packages that are not in the standard library?

https://github.com/kubernetes/kubernetes/blob/92285fd74e8906bc4910d6fb6729b7ec0e74f427/pkg/kubelet/pod_workers.go#L19-L39

https://github.com/kubernetes/kubernetes/blob/92285fd74e8906bc4910d6fb6729b7ec0e74f427/pkg/kubelet/volume_host.go#L19-L46

Reply:
as far as I know it should be 2 sections: stdlib and other packages. Not sure the linter catches this every time

Reply:
+1. The most common pattern I have seen in kubernetes is 2 sections, excluding few files that have the 3 sections. I'm not sure if this can actually be mandated using a linter. We can keep the 2 sections as is in this file and just move the new packages to other packages section.